sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1010,7 +981,7 @@ def _yield_region_zones(self, to_provision: resources_lib.Resources,
             # Check the *previous* cluster status. If the cluster is previously
             # stopped, we should not retry other regions, since the previously
             # attached volumes are not visible on another region.
-            if cluster_status == global_user_state.ClusterStatus.STOPPED:
+            if prev_cluster_status == global_user_state.ClusterStatus.STOPPED:
```

Comment:
L978-980 seems to describe L1000?

Reply:
It should be go before the `prev_cluster_status == UP`. I moved it and rephrased it for clarity. : )